crates/holochain/src/core/ribosome/host_fn/delete_link.rs

```diff
@@ -39,7 +39,7 @@ pub fn delete_link<'a>(
                     let workspace = call_context_2.host_context.workspace();
                     CascadeResult::Ok(
                         CascadeImpl::from_workspace_and_network(&workspace, network)
-                            .dht_get(address_2.into(), GetOptions::local())
+                            .dht_get(address_2.into(), GetOptions::network())
```

Comment:
Should this be exposed to the app developer like we do in other functions? I think it's a valid fix and apps that need to work with 0 arc really aren't going to work well offline without a whole lot of work yet - but maybe it should still be the app developer's choice?

Reply:
Yes, better unify this API.

Reply:
How about now?

Reply:
Yes, great!